source/scpd/Cpp.d

```diff
@@ -69,7 +69,13 @@ private mixin template CPPBindingMixin (T, bool Copyable = true, bool DefaultCon
             dtorCPPObject!T(&this);
         }
 
-        @disable this (this);
+        // Workaround for https://issues.dlang.org/show_bug.cgi?id=22505
+        static if (Copyable)
+        this (this) @trusted nothrow @nogc pure
+        {
+            T temp = T(this);
+            copyCtorCPPObject!T(&this, &temp);
+        }
```

Comment:
Waait what ? We have a copy ctor, a postblit should never be called.

Reply:
`druntime` works in mysterious ways

Reply:
Do you know what calls it ?

Reply:
Dynamic array runtime, while trying to relocate the array.